build.rs

```diff
@@ -255,3 +255,10 @@ fn platform_is_x64() -> bool {
 fn platform_is_s390x() -> bool {
     env::var("CARGO_CFG_TARGET_ARCH").unwrap() == "s390x"
 }
+
+fn platform_is_emulated_aarch64() -> bool {
+    env::var("CARGO_CFG_TARGET_ARCH").unwrap() == "aarch64"
+        && env::var("CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUNNER")
```

Comment:
Oh we've actually already [got another env var](https://github.com/bytecodealliance/wasmtime/blob/fb07ff574028e392395329a5e68f9f231b929981/.github/workflows/main.yml#L271-L273) for indicating that qemu is in use and virtual memory should be limited as a result, perhaps that would suffice for this test too?

Reply:
Thanks, checking it instead should be more robust in case we decide to test on other platforms with QEMU.